src/components/Dropdown/Dropdown.tsx

```diff
@@ -228,6 +229,7 @@ export default class Dropdown extends AutoControlledComponent<
       focused: false,
       searchQuery: search ? '' : undefined,
       value: multiple ? [] : null,
+      defaultHighlightedIndex: !this.props.search && !this.props.multiple ? null : undefined,
```

Comment:
Can we document with some comment why we use null and why undefined?

Reply:
added one. is it ok?